src/main/java/org/snomed/snowstorm/core/data/services/identifier/IdentifierCacheManager.java

```diff
@@ -194,6 +216,7 @@ public void stopBackgroundTask() {
 		stayAlive = false;
 		cacheDaemon.interrupt();
 		cacheDaemon = null;
+		identifierCaches.clear();
```

Comment:
Do we need this? Doesn't that kill the caches?

Reply:
It does kill the cache. I made the change as the cache in the tests were overlapping. Because the method is only called from the shutdownPolling method, I figured it was fine to do as that method itself is called when the bean is no longer required.